### PR TITLE
:art: :chart_with_upwards_trend:  :triangular_ruler: Update Assessment Results Page Layout and Design #530

### LINKED ISSUE #530: 🎨 📈 📐 Update Assessment Results Page Layout and Design
## Technologies and Knowledge Scope 🧠

Please delete options that are not relevant.

- [ ] HTML, SCSS
- [ ] Typescript

## User stories

Update the styling and layout of the assessment results page to match the below design:

MicrosoftTeams-image (6)

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Description

The resulting update should match the proposed design and update the position of the different components in the results page:

- The results table
- The placement and design of "Edit" and "Go back" buttons
- The placement and design Search Bar. Tip: You can re-use the design of the seach bar on the bots list page. Dashboard -> Active Bots
- Add the insights section. Use the below image placeholder for the insights section:
image

- Add the pie chart section: Use the below image placeholder for the pie chart: 
image

### Acceptance Criteria

1. The layout should follow the structure and placement of components in the design.
2. The resulting implementation should not include an implementation or styling of any of the new components e.g. insights and pie chart. (This will be done in separate issues).
3. The resulting implementation should have place holders in place of:
    i. The insights section
    ii. The piechart section

### PR DESCRIPTION
**Pull request by lilmithi**
# Description

- This PR resolves issue #530. 
- Image placeholders and the edit assessment button have been added in this issue.

Fixes #530 

# Screenshot
image

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ReaganCn** [comment]
Good work @lilmithi. I can't see the changes from the link provided. Can you see them from your side.

File: libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html [hunk 1]
```diff
@@ -1,66 +1,210 @@
...
+<convl-page title="{{ pageTitle }}" fxLayout="column" fxFlexFill>
   <div class="page-container">
     <div class="title-wrapper">
-      <button class="btn back-btn" (click)="goBack()">{{'ASSESSMENTS.RESULTS.BUTTONS.GO-BACK' | transloco}}</button>
+      <button class="btn back-btn" (click)="goBack()">
+        {{ 'ASSESSMENTS.RESULTS.BUTTONS.GO-BACK' | transloco }}
+      </button>
+      <button class="btn edit-assessment-btn">
+        <mat-icon>edit</mat-icon>
+        Edit assessment
```
Comment (ReaganCn) [inline libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html:9]: Use multi-lang

File: libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html [hunk 2]
```diff
@@ -1,66 +1,210 @@
...
+          src="https://user-images.githubusercontent.com/34684844/247146711-c32646a0-4f35-4d1c-8e30-1c211acf6503.png"
+          alt="insights placeholder"
+          style="width: 100%"
+        />
+      </div>
+      <div class="chart-wrapper">
+        <div class="search-results-wrapper">
+          <span style="font-size: 1.1rem">Science Assessment Results</span>
+          <select name="assessment results">
+            <option value="">All classes</option>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html:24]: Here too

File: libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html [hunk 3]
```diff
@@ -1,66 +1,210 @@
...
+      <div class="cards-wrapper" style="flex: 1">
+        <img
+          src="https://user-images.githubusercontent.com/34684844/247146711-c32646a0-4f35-4d1c-8e30-1c211acf6503.png"
+          alt="insights placeholder"
+          style="width: 100%"
+        />
+      </div>
+      <div class="chart-wrapper">
+        <div class="search-results-wrapper">
+          <span style="font-size: 1.1rem">Science Assessment Results</span>
```
Comment (ReaganCn) [inline libs/features/convs-mgr/conversations/assessments/src/lib/pages/assessment-results/assessment-results.component.html:22]: This one too

**Review by ReaganCn — CHANGES_REQUESTED** [review]
Just some minor changes

**Comment by lilmithi** [comment]
Requested changes have since been implemented. Kindly requesting review

**Review by ReaganCn — APPROVED** [review]

**Pull request merged**